libs/vre/shared/app-resource-properties/src/lib/properties-display.component.ts

```diff
@@ -42,10 +42,15 @@ import { sortByKeys } from './sortByKeys';
     </div>
 
     <!-- list of properties -->
-    <ng-container *ngIf="myProperties$ | async as myProperties">
-      <ng-container *ngIf="myProperties.length > 0; else noProperties">
+    <ng-container>
+      <ng-container *ngIf="properties.length > 0; else noProperties">
         <app-property-row
-          *ngFor="let prop of myProperties; let last = last; trackBy: trackByPropertyInfoFn"
+          [ngClass]="
+            (showAllProperties$ | async) || (prop.values && prop.values.length > 0)
```

Comment:
try to reduce code duplication by introducing an observable method which returns the class name

Reply:
Jup, of course.